bookings/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db       import models
+from users.models    import User
+from products.models import Product
+
+class Booking(models.Model):
+    user         = models.ForeignKey(User, on_delete=models.CASCADE)
+    product      = models.ForeignKey(Product, on_delete=models.CASCADE)
+    check_in     = models.DateField
+    check_out    = models.DateField
+    head_count   = models.IntegerField
```

Comment:
제대로 동작하는지 확인 해보셨나요~?

```suggestion
    check_in     = models.DateField()
    check_out    = models.DateField()
    head_count   = models.IntegerField()
```

Reply:
`created_at`, `updated_at` 정보는 데이터 관리자 입장에서 매우 중요한 데이터입니다. 추가해주세요~